docs/how_to/tensorflow_install/tensorflow_install.md

```diff
@@ -33,9 +33,7 @@ Follow these steps:
 2. Once you have pulled the image, run it by using the command below:
 
    ```bash
-   docker run -it --network=host --device=/dev/kfd --device=/dev/dri
-   --ipc=host --shm-size 16G --group-add video --cap-add=SYS_PTRACE
-   --security-opt seccomp=unconfined rocm/tensorflow:latest
+   docker run -it --network=host --device=/dev/kfd --device=/dev/dri --ipc=host --shm-size 16G --group-add video --cap-add=SYS_PTRACE --security-opt seccomp=unconfined rocm/tensorflow:latest
```

Comment:
Instead of combining into a single line, it would be better to keep the original 3 separate lines and use the \ to signify a line break in bash. If you wish to correct it, please do so or I can make an alternate PR.

Reply:
Thank you for pointing out this issue.

Reply:
@amathews-amd FYI. Please review.

Reply:
Ok, I've followed your advice @saadrahim 

Thank You..